.circleci/config.yml

```diff
@@ -16,33 +19,59 @@ jobs:
           paths:
             - ~/.m2
           key: client-side-events-{{ checksum "pom.xml" }}
-          
-  "deploy_prod":
+
+  deploy_stage:
```

Comment:
@adituriya is it safe to deploy to staging? It seems the target project is always `client-side-events`.

Reply:
@fjvallarino this project doesn't really have a "staging" environment, so we decided to deploy to `client-side-events` and test the version manually. I'm not sure if we should revert that or keep it for future use. Thanks.